packages/services/test/src/kernel/kernel.spec.ts

```diff
@@ -146,20 +153,16 @@ describe('kernel', () => {
       expectFailure(kernelPromise, done, '');
     });
 
-    it('should auto-reconnect on websocket error', (done) => {
+    it('should auto-reconnect on websocket error', async () => {
```

Comment:
Is there a special symbol or something on async functions such that the tester is able to identify it as such? I was under the impression it used `args.length` with the `done` function,

Reply:
async functions return a promise - that's how the testing framework knows what it is.